TWLight/applications/views.py

```diff
@@ -767,6 +767,34 @@ class EvaluateApplicationView(NotDeleted, CoordinatorOrSelf, ToURequired, Update
     success_url = reverse_lazy('applications:list')
 
     def form_valid(self, form):
+        app = self.object
+        status = form.cleaned_data['status']
+        if app.partner.authorization_method == Partner.PROXY:
+            total_accounts_available_for_distribution = None
+            if app.specific_stream is not None:
+                if app.specific_stream.accounts_available is not None:
+                    total_apps_approved_or_sent = User.objects.filter(
```

Comment:
We do this object filtering a few times, both here from this PR and also in other views (partner detail?). Perhaps we should move it to a helper function that takes an application or partner and returns total apps approved or sent?